### PR TITLE
Add Thanos proposal to SoD

### PR DESCRIPTION
**Pull request by povilasv**
REF https://github.com/thanos-io/thanos/issues/2041

### CODE REVIEW

**Comment by bwplotka** [comment]
:+1:

File: seasonofdocs/README.md [hunk 1]
```diff
@@ -103,4 +103,36 @@ The Kubernetes special interest group (SIG) for command line interfaces (CLI), o
 
 - How to work with stakeholders in open source projects
 - How to document a new tool
-- Bleeding edge command line tools for Kubernetes
\ No newline at end of file
+- Bleeding edge command line tools for Kubernetes
+
+Summer of docs
```
Comment (bwplotka) [inline seasonofdocs/README.md:108]: Do we need this?

File: seasonofdocs/README.md [hunk 2]
```diff
@@ -103,4 +103,36 @@ The Kubernetes special interest group (SIG) for command line interfaces (CLI), o
...
+
+### Thanos: Complete Katacoda tutorials
+
+This is a great project for a writer with some passion for Docker and Kubernetes, interested in doing interactive docs: specifically learning to build [Katacoda Course](https://katacoda.com/scenario-examples/scenarios/create-course).
```
Comment (bwplotka) [inline seasonofdocs/README.md:112]: Let's add internal issue link and who is the mentor WDYT? @povilasv

Comment (povilasv) [reply seasonofdocs/README.md:110]: Added in https://github.com/cncf/mentoring/pull/193/files#diff-c29d1af42af4082493795ee8f5ccbcf2R130

**Pull request merged**